config/defaults.conf

```diff
@@ -140,6 +140,9 @@ batch_size = 32  // Training batch size.
 optimizer = adam  // Optimizer. All valid AllenNLP options are available, including 'sgd'.
                   // Use 'bert_adam' for reproducing BERT experiments.
                   // 'adam' uses the newer AMSGrad variant.
+                  // Warning: bert_adam is designed for cases where the number of epochs is known
```

Comment:
Are we sure it's still `bert_adam`? AFAIK they updated it in `pytorch_transformers`, but not sure if AllenNLP updated.

Reply:
Yep—we're still using AllenNLP tools for optimization, so it's still bert_adam.